.github/workflows/documentation-checks.yaml

```diff
@@ -29,18 +29,18 @@ on:
 jobs:
   docchecks:
     name: Run documentation checks
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
```

Comment:
should we still run on such an old version of Ubuntu?

Reply:
This was requested by @rkratky in #7 - has it changed by now, Robert?

Reply:
Thanks @ru-fu for including that PR.

Reply:
The kteam-docs is using 22.04 also; we don't have any complaints specific to the Ubuntu version so far, if that helps.

Reply:
would it help if i ran the github workflow again with 20.04 and 22.04 to get the run time comparison?
even though the default workflow for our starter pack is relatively simple since it doesn't do anything complicated besides building the docs.